hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -203,6 +205,93 @@ class AzkabanHelper {
   }
 
   /**
+   * Converts epoch to Date format.
+   *
+   * @param epoch timestamp
+   * @return Date format in string. Returns "- "if epoch is -1
+   */
+  static String epochToDate(String epoch) {
```

Comment:
Can we move the new methods in this class to a helper class in the azkaban-cli project?